test/tasks/processIssueComment/utils/parse-comment/index.test.js

```diff
@@ -0,0 +1,257 @@
+            contributors: {
+                'jane.doe23': ['doc', 'infra'],
+            },
+        })
+    })
+
+    test('Support split words (like user testing)', () => {
+        expect(
+            parseComment(
+                `@${testBotName} please add jakebolam for infrastructure, fund finding`,
+            ),
+        ).toEqual({
+            action: 'add',
+            contributors: {
+                jakebolam: ['infra', 'fundingFinding'],
+            },
+        })
+
+        expect(
+            parseComment(
+                `@${testBotName} please add jakebolam for infrastructure, user testing and testing`,
+            ),
+        ).toEqual({
+            action: 'add',
+            contributors: {
+                jakebolam: ['infra', 'userTesting', 'test'],
+            },
+        })
+    })
+
+    test('Support split words types that are referenced via other terms (e.g. a plural split word)', () => {
+        expect(
+            parseComment(
+                `@${testBotName} please add @jakebolam for infrastructure, funds`,
+            ),
+        ).toEqual({
+            action: 'add',
+            contributors: {
+                jakebolam: ['infra', 'fundingFinding'],
+            },
+        })
+    })
+
+    // test('Add multiple users in 1 hit - for some contributions', () => {
+    //     expect(
+    //         parseComment(
+    //             `@${testBotName} please add @jakebolam and @tbenning for doc and review`,
+    //         ),
+    //     ).toEqual({
+    //         action: 'add',
+    //         contributors: {
+    //             jakebolam: ['doc', 'review'],
+    //             tbenning: ['doc', 'review'],
+    //         },
+    //     })
+    // })
+
+    test('Add multiple users in 1 hit - seperate sentences', () => {
+        expect(
+            parseComment(
+                `@${testBotName} add @kazydek for doc, review, maintenance. Please add akucharska for code, design. Please add derberg for infra and ideas`,
+            ),
+        ).toEqual({
+            action: 'add',
+            contributors: {
+                kazydek: ['doc', 'review', 'maintenance'],
+                akucharska: ['code', 'design'],
+                derberg: ['infra', 'ideas'],
+            },
+        })
+    })
+
+    // TODO: Looks like this is gramatically incorrect
+    // test('Add multiple users in 1 hit - sentences seperated by commas :think:', () => {
+    //     expect(
+    //         parseComment(
+    //             `@${testBotName} please add kazydek for doc, review, maintenance, please add akucharska for code, maintenance and please add derberg for doc, ideas`,
+    //         ),
+    //     ).toEqual({
+    //         action: 'add',
+    //         contributors: {
+    //             kazydek: ['doc', 'review', 'maintenance'],
+    //             akucharska: ['code', 'maintenance'],
+    //             derberg: ['doc', 'ideas']
+    //         },
+    //     })
+    // })
```

Comment:
@gr2m @Berkmann18 do we need to include this one? I think this is an invalid test for me. Let me know your thoughts about this. Thanks.

Reply:
I'd like to wait for @Berkmann18 to comment, he knows more about the natural language processing

Reply:
> I'd like to wait for @Berkmann18 to comment, he knows more about the natural language processing

@gr2m ok sure 😄 there are only 2 tests left then we can merge this one 🚀

Reply:
@Berkmann18 please check and let me know. Also, if you want to add other tests we need for this one. Cheers! 🎉

Reply:
@Berkmann18 gentle ping

Reply:
okay let's go ahead and merge, we can revert if Max finds any problems

Reply:
> okay let's go ahead and merge, we can revert if Max finds any problems

@gr2m all right, let's merge this one 🚀